frontend/src/features/projects/common/components/columns.tsx

```diff
@@ -259,7 +259,12 @@ export const getPropertyColumns = ({
       width: 60,
       clickable: true,
       Cell: ({ cell: { value } }: CellProps<IProperty, number>) => {
-        const icon = value === 0 ? <LandSvg title="Land" /> : <BuildingSvg title="Building" />;
+        const icon =
+          value === 0 ? (
+            <LandSvg title="Land" className="svg" />
```

Comment:
Had to use the className instead of defaulting all `svg` elements to have default css.  I didn't have time to fix all svg elements to work better with consistent styling.